### PR TITLE
Chore/code cleanup

### PR DESCRIPTION
**Pull request by joncojonathan**
Corrected some typos in the code while also improving efficiency and clarity.  Hope this helps.

### CODE REVIEW

File: src/Client.php [hunk 1]
```diff
@@ -84,16 +85,16 @@ public function setQrcodeCallback($closure)
...
             $this->login();
         }
         $this->WsapiWrapper = new Wrapper($this->client);
     }
 
     public function loop($callback)
     {
-        $this->running = true;
-        while ($this->running) {
+        $running = true;
```
Comment (vitormattos) [inline src/Client.php:96]: this is necessary to stop the execution of client

File: src/Client.php [hunk 2]
```diff
@@ -84,16 +85,16 @@ public function setQrcodeCallback($closure)
...
+        while ($running) {
             $jsMessages = $this->WsapiWrapper->getBufferedNewMessages();
             if ($jsMessages) {
                 $callback($this, $jsMessages);
```
Comment (vitormattos) [inline src/Client.php:100]: The client instance is passed to the callback, if the callback marks "stop" on the client, the loop should be interrupted

Comment (joncojonathan) [reply src/Client.php:100]: Looking at this I have no idea why I changed that.  I'll put it back shortly.  Apologies.

Comment (joncojonathan) [reply src/Client.php]: I've now reverted that change, apologies again.

File: src/Client.php [hunk 3]
```diff
@@ -21,7 +23,6 @@ class Client
     private $sessionFile;
     private $qrcodeCallback;
     private $logged = false;
-    private $running = false;
```
Comment (vitormattos) [inline src/Client.php:24]: This property is necessary to control the while loop

**Review by vitormattos — CHANGES_REQUESTED** [review]

File: src/Client.php (hunk 3 above)
Comment (joncojonathan) [reply src/Client.php]: I've reverted that change.  Could you explain why it cannot be changed to a local variable (as I did originally) as I don't understand.  Many thanks.

**Comment by joncojonathan** [comment]
Hello, I think this is now fixed?

**Review by vitormattos — APPROVED** [review]

**Comment by vitormattos** [comment]
Only need wait for CI

**Comment by joncojonathan** [comment]
CI seems to pass now.

**Pull request merged**